### PR TITLE
Refactoring documentation and adding meeting practices

### LINKED ISSUE #143: Proposed structure for 1-on-1 syncs
# Summary

I had my first 1-on-1 chat with @choldgraf on Friday and I'm sure many of you will have yours upcoming if you've not had one already. During that meeting, I shared with Chris my workflow for checking in with myself on how my work is progressing and I also apply this in various other mentoring networks I'm involved in. I thought I'd share and see if this is a structure we'd like to trial within the team. This workflow is heavily inspired by Kirstie Whitaker's Lab, the full process for which, including meeting template, can be viewed here: https://github.com/WhitakerLab/Onboarding/blob/master/Setting-up-your-weekly-meetings.md

This is me sharing my workflow and proposing we try it out. I'm open to tweaks, discussions and alternative ideas - and I absolutely don't think this is the one right way to have these sorts of meetings!

(I'm going to be using the terms mentee and mentor to describe the roles in this setup, as opposed to say junior team member vs senior team member. While that is likely the dynamic that will emerge in these meetings, I also believe we are all capable of learning from one another, regardless of our levels of experience - and it's the fact that they're different that can supercharge these types of knowledge exchanges!)

### Meeting template

The meeting template involves answering the following questions (bullet points are fine!) - which you'll notice heavily overlap with our team sync! 😉 

- Who did you help this week/fortnight/...?
- Who helped you this week/fortnight/...?
- What did you achieve this week/fortnight/...?
- What did you struggle with this week/fortnight/...?
- What do you plan to work on next week/fortnight/...?
- Where do you need extra help?
- Anything other topics

This doesn't need to be stuck to rigidly, rather think of it as a way to structure your thoughts around how you've been progressing.

### Meeting structure

1. The mentee prepares their answers to the above questions ahead of the meeting
2. Mentee and mentor go through the questions with a discussion and try to solve any blockers and celebrate any achievements

In my experience, this has happened in public repositories. This provides some nice book-ending in terms of a pull request being opened before a meeting, and a pull request merged when mentor and mentee are satisfied. You can see some of my notes from meetings I have with my mentor in Kirstie's lab here: https://github.com/WhitakerLab/WhitakerLabProjectManagement/pull/510. So we could do this in the team-compass repo, or the meta repo if folks don't feel comfortable with this being entirely public.

### What about privacy / sensitive topics?

While doing this sort of team interaction in the open is inline with 2i2c's transparency as an org, it can be off-putting for individuals to bring up sensitive discussion points.

First and foremost, **the meeting will always be private** and the above template doesn't need to be stuck to rigidly. Again, it's a tool to help us structure our thoughts. Other techniques I've implemented to maintain privacy is intentionally keeping updates in the Markdown vague and using the meeting to go into greater detail.

Everyone on the team will have different risk levels for this so I'm happy to keep this open for discussion and hear alternative ideas 🙂 

# Actions

- [x] Discuss above proposal
- [x] Scope what we do and don't want these types of meetings to be
- [x] Agree on a draft 1-to-1 meeting structure
- [ ] Implement in team compass https://github.com/2i2c-org/team-compass/pull/164

### PR DESCRIPTION
**Pull request by choldgraf**
This refactors a bit of our team compass documentation with the goals of:

- De-cluttering the site navigation, so that there aren't so many top-level links
- Re-grouping sections into more natural ways
- Providing some top-level sections with nested sub-sections that could be more naturally extended
- Breaking up longer pages into more shorter pages
- Adding some extra content around team practices (e.g. for meetings, and updating some practices that weren't updated from previous PRs)

closes #143

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]
I like it! I just had a click around in the top level on the RTD build, but looks a lot cleaner

File: get-started.md [hunk 1]
```diff
@@ -78,3 +61,39 @@ Check out the [projects page](reference/projects.md) for more information about
...
+Generally speaking the daily specifics of activity on a project are recorded *outside* of the team compass, as they are more dynamic. The important thing is that the team compass make it clear *where* to find this information, and that the information is kept up-to-date for all.
+
+
+## Marketing and communication
+
+There are three main ways that 2i2c communicates with the outside world:
+
+- Twitter: [@2i2c_org](https://twitter.com/2i2c_org)
+  - Currently, there is nobody activately monitoring the Twitter account. We use it to post links to blog posts or major announcements in the newsletter.
+- Newsletter: [using Mailchimp](https://mailchimp.com/)
```
Comment (damianavila) [inline get-started.md:91]: How frequently do we send newsletters and what do we usually say there? Do we have records of previous newsletters?

**Review by damianavila — APPROVED** [review]
Just a minor question but not a blocker.
LGTM.

File: get-started.md (hunk 1 above)
Comment (choldgraf) [reply get-started.md:91]: mostly it is just mirroring our blog posts - a way to get the information to a more dedicated group of folks.

**Comment by choldgraf** [comment]
ok merging this in so that we can iterate elsewhere as well!

**Pull request merged**